etcdctlv3/command/range_command.go

```diff
@@ -48,13 +58,40 @@ func rangeCommandFunc(cmd *cobra.Command, args []string) {
 	if err != nil {
 		ExitWithError(ExitError, err)
 	}
+
+	sortByOrder := pb.RangeRequest_NONE
+	switch {
+	case sortOrder == "ASCEND":
+		sortByOrder = pb.RangeRequest_ASCEND
+	case sortOrder == "DESCEND":
+		sortByOrder = pb.RangeRequest_DESCEND
```

Comment:
probably ErrBadFlag. But I do not want to create so many error types either. Can you check what other command line tool does? Like curl.